codersdk/deployment.go

```diff
@@ -55,6 +58,11 @@ const (
 	FeatureControlSharedPorts         FeatureName = "control_shared_ports"
 )
 
+var AcceptedMetricAggregationLabels = []string{
+	agentmetrics.TemplateNameLabel, agentmetrics.AgentNameLabel,
+	agentmetrics.UsernameLabel, agentmetrics.WorkspaceNameLabel,
+}
```

Comment:
suggestion: expose `agentmetrics.LabelAll []string` and then just import that here; if we end up adding more labels here it's more likely that we'll forget to handle it here due to the non-locality.